metadata/nano_UL17_test.json

```diff
@@ -0,0 +1,8 @@
+{
+"TTTo2L2Nu_TuneCP5_13TeV-powheg-pythia8": [
```

Comment:
see comment in the workflow yml files: might be better to have a small version of these samples (maximum 1000 events) on btv eos and delete the `test_w_dj_*.json` files. Note that we would need again a double muon version etc of this json file

Reply:
same comments in [above](https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/50#discussion_r1094747204)